docs/file-reference.md

```diff
@@ -30,6 +30,8 @@ services:
   - port: 8080
     targetPort: 80
     endpoint: minikube.external/foo
+  portMappings:
```

Comment:
this doesn't make sense in this example, as there is nothing running on port 8081

Reply:
updated

Reply:
I don't know if this makes sense, why would you expose the same port twice?

I think that all our examples like this have to work and make sense. 

I don't think that showing one example with all Kedge keys has any benefit.

Reply:
I agree, but I think we should cover all the fields in file-reference.md so we can show them in use. Where do you suggest we add that?
I think we should have an example which shows all the fields added/modified by kedge in file-reference, it does not have to be a working example, but something that has all the fields and acts as a quick reference on how and where to use that field.

As for this, should I remove this?

Reply:
I think that if we show an example with all the fields it will be just hard to read and it won't provide any benefits.

I think that far better is to have multiple examples highlighting given shortcut in the scenario that is best for it. We should show how it should be used, in the scenarios we think is the best for given shortcut and highlight our intent behind it.

Having multiple examples in files-reference.md will be also mess, so we can just reference examples in docs/examples/ 

In this case, we can just link to docs/examples/portMappings/httpd.yaml  with some short explanation.

Reply:
done